androidapp/DansFrappesRaps/app/src/main/java/com/cs3450/dansfrappesraps/ui/viewmodels/CartScreenViewModel.kt

```diff
@@ -111,8 +111,23 @@ class CartScreenViewModel(application: Application): AndroidViewModel(applicatio
         return uiState.priceSum
     }
 
-    suspend fun checkInventory() {
-
+    fun checkInventory(){
```

Comment:
The price needs to be calculated based off the cost of all the ingredients of EACH drink in the cart. As far as I can tell this is only doing it based off one.

Reply:
We also need to account for quantity of the drinks. There should be a total count of the correct number of drinks in the cart.